modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryUtils.java

```diff
@@ -131,6 +133,32 @@
     /** */
     private static final Set<Class<?>> SQL_TYPES = createSqlTypes();
 
+    /** Default SQL delimeter. */
+    public static final char DEFAULT_DELIM = '\n';
```

Comment:
can you use U.nl() here ? yes i see that it returns String instead of char.

Reply:
`\n` character used right now. 
You can see it from the diff.
I don't think this PR is the right place to change existing formatting.